README.md

```diff
@@ -1,2 +1,40 @@
 # techteamberlin_backend_challenge
```

Comment:
Generally we would like you to add automatic tests since we require a good test coverage when delivering real-world projects.

Reply:
I have added some test, to test all end point. It was little bit challenge for me, since honestly familiar with Test Driven Development. But I did manage to learn some of it concept, and applied the feedback. I would say that the test I wrote are not perfect, maybe the more I will continue to use TDD, the more reliable test I will be write.